src/xSTIR/cSTIR/include/sirf/STIR/stir_x.h

```diff
@@ -910,6 +910,10 @@ The actual algorithm is described in
         {
             stir::ScatterEstimation::set_mask_proj_data_sptr(arg->data());
         }
+		void set_recompute_mask_image(bool arg)
+        {
+            stir::ScatterEstimation::set_recompute_mask_image(arg);
+        }
```

Comment:
```suggestion
#if STIR_VERSION >= 060200
        void set_recompute_mask_image(bool arg)
        {
            stir::ScatterEstimation::set_recompute_mask_image(arg);
        }
#endif
```
Otherwise we will get trouble with older STIR.